src/richText.js

```diff
@@ -411,13 +411,15 @@ define([
             replacer, result;
         if (escape) {
             replacer = function () {
-                var id = util.get_guid();
-                places[id] = replacePathWithBubble(form, this.outerHTML);
+                var id = util.get_guid(),
+                    text = xml.humanize($("<div>").append($(this).clone()));
```

Comment:
Ah, further testing shows that I should be using `xml.normalize()` instead of `xml.humanize()`, which makes sense since ckEditor is an HTML editor, not a plain text editor. New PR coming.